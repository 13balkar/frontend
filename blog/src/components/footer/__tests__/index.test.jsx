import { render, screen } from '@testing-library/react';
import * as React from 'react';
import Footer from '..';
describe('Footer', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
  it('should render correctly with Footer data', () => {
    render(<Footer/>);
    expect(screen.getByText('Â©artifact.com 2019')).toBeInTheDocument();
  });
});
