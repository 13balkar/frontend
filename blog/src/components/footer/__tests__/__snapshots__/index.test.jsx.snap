// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<DocumentFragment>
  <footer
    class="outerPadding"
  >
    <p>
      Â©artifact.com 2019
    </p>
  </footer>
</DocumentFragment>
`;
